config/Version.internal.xcconfig

```diff
@@ -1,9 +1,4 @@
 VERSION_SHORT=19.0
 
 // Internal long version example: VERSION_LONG=9.9.0.20180423
-//
-// Technically, the date for this version was 10/01, but this is the second
-// build for the day so I bumped it to 10/02. On 10/04 we'll code freeze 18.4
-// and this should be the very last build for 18.3, so we should be fine.
-// – Gio
-VERSION_LONG=19.0.0.20220121
+VERSION_LONG=19.0.0.20220122
```

Comment:
> Also notice that I bumped the internal version to "...22", which
technically would mean the build was made on the 22nd of January, even
though (as far as UTC goes) we're still on the 21st. Pragmatically, that
has little consequence because we'll most likely won't be making any
build tomorrow.

Can't wait for us to address https://github.com/wordpress-mobile/release-toolkit/issues/202 one day…